com.unity.netcode.gameobjects/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -409,10 +412,24 @@ public void TrackSceneEventReceived(ulong senderClientId, uint sceneEventType, s
 
         public void TrackPacketSent(uint packetCount)
         {
```

Comment:
I would make this plural as well as `TrackPacketsSent` for the same reason mentioned above.

Reply:
Now that you're using counters rather than events I think this has become clearer and is no longer necessary